gd/memory/interface.py

```diff
@@ -493,6 +493,18 @@ def enable_object_limit(self) -> None:
         self.write_bytes(Buffer[0x80, 0x38, 0x01, 0x00], 0x87B17)
         self.write_bytes(Buffer[0x80, 0x38, 0x01, 0x00], 0x880F4)
         self.write_bytes(Buffer[0x80, 0x38, 0x01, 0x00], 0x160B06)
+    def enable_anticheat(self) -> None:
+        self.write_bytes(Buffer[0xEB2E], 0x202AAA); self.write_bytes(Buffer[0x742E], 0x202AAA) # Speedhack kicks
+        self.write_bytes(Buffer[0xEB], 0x15FC2E); self.write_bytes(Buffer[0x74], 0x15FC2E) # Editor kick
+        self.write_bytes(Buffer[0xEB0C], 0x1FD557); self.write_bytes(Buffer[0x740C], 0x1FD557) # Level completion kicks
+        self.write_bytes(Buffer[0x90, 0x90, 0x90, 0x90, 0x90, 0x90], 0x1FD742); self.write_bytes(Buffer[0x80, 0xBF, 0xDD, 0x02, 0x00, 0x00, 0x00, 0x0F, 0x85, 0x0A, 0xFE, 0xFF, 0xFF, 0x80, 0xBF, 0x34, 0x05, 0x00, 0x00, 0x00, 0x0F, 0x84, 0xFD, 0xFD, 0xFF, 0xFF], 0x1FD742)
```

Comment:
This line looks strange, since amount of bytes in each part differ. Check this one once again.